components/WeatherForecastModal.js

```diff
@@ -0,0 +1,92 @@
+import {
+  WeatherForecastModalField,
+  ModalCloseButton,
+  Table,
+  DisplayedDaysContainer,
+  DisplayedDay,
+  Date,
+  Daytime,
+  WeatherDataContainer,
+  WeatherIcon,
+  Temperature,
+  Windspeed,
+} from "./styledComponents/WeatherForecasModal.styles";
+import WeekdayFromDateString from "./WeekdayFromDateString";
+import ArrowIcon from "./icons/ArrowWeatherIcon";
+
+export default function WeatherForecastModal({
+  filteredWeatherMorning,
+  filteredWeatherAfternoon,
+  filteredWeatherEvening,
+  onClickButton,
+}) {
+  // extracting the dates from the date-time-String with help of the static end value "06:00:00"
+  const dates = filteredWeatherMorning
+    .filter((date) => date.dt_txt.endsWith("06:00:00"))
+    .map((date) => date.dt_txt.split(" ")[0]);
+
+  return (
+    <>
+      <WeatherForecastModalField>
+        <ModalCloseButton onClick={onClickButton}>x</ModalCloseButton>
+        <Table>
+          <tbody>
+            <td>
+              {/*because of space issues and of the everchanging WeatherArray, the filteredWeatherArrays will only be mapped to the third day  */}
+              {filteredWeatherMorning.slice(0, 3).map((weekday, index) => (
+                <DisplayedDaysContainer key={index}>
+                  <DisplayedDay>
+                    <WeekdayFromDateString dateString={weekday.dt_txt} />
+                  </DisplayedDay>
+                  <Date>{dates[index]}</Date>
+                </DisplayedDaysContainer>
+              ))}
+            </td>
+            <td>
+              <Daytime>6am</Daytime>
+              {filteredWeatherMorning.slice(0, 3).map((weather, index) => (
```

Comment:
I I see this correctly you only ever use the first 3 days of your forecast. Isn't it possible to slice the data from the API directly (`weather.list.slice(0,3)`)? Then you don't need to repeat this here over and over again.